plugins/modules/ipauser.py

```diff
@@ -573,6 +584,14 @@
     name: pinky,brain
     state: enabled
 
+# Remove but preserve user pinky
```

Comment:
We can remove several users without resorting to `users` simply by adding multiple names. In my opinion, that's a more readable way to do this.

@t-woerner what do you think?

Reply:
Yes, I agree. It is working both ways, though. Therefore it might be good to have both examples.